File: Python/4/2.py
# Get input from user
num = int(input("Enter a number: "))

# Check if string of num is same even if reversed
if str(num) == str(num)[::-1]:
    print("{0} is a palindrome".format(num))
else:
    print("{0} is not palindrome".format(num))

File: C/2/2.c
#include <stdio.h>

int a;

int main()
{
    printf("Enter a number: ");
    scanf("%d", &a);

    if (a % 2 == 0)
    {
        printf("\n%d is an even number\n", a);
    }
    else
    {
        printf("\n%d is an odd number\n", a);
    }

    return 0;
}
